modP2pImpl/src/org/aion/p2p/impl/P2pMgr.java

```diff
@@ -717,23 +504,27 @@ public void run() {
             selector = Selector.open();
 
             scheduledWorkers = new ScheduledThreadPoolExecutor(1);
-            workers = Executors.newFixedThreadPool(Math.min(Runtime.getRuntime().availableProcessors() * 2, 16), new ThreadFactory() {
-
-                private AtomicInteger cnt = new AtomicInteger();
-
-                @Override
-                public Thread newThread(Runnable r) {
-                    return new Thread(r,"p2p-worker-" + cnt.incrementAndGet());
-                }
-            });
+            workers = Executors.newFixedThreadPool(Math.min(Runtime.getRuntime().availableProcessors() * 2, 16));
+            // workers = Executors.newCachedThreadPool();
```

Comment:
Why we stop naming IO threads?

Reply:
Oops, this was a mistake on my part please add it back